src/main/java/org/takes/facets/fallback/TkFallback.java

```diff
@@ -75,23 +76,36 @@ public Response act(final Request req) throws IOException {
      */
     private static Response route(final Take take, final Fallback fbk,
         final Request req) throws IOException {
+        // @checkstyle LineLength (1 line)
+        final String error = "There is no fallback available. The original problem is this: %s";
```

Comment:
@danff why this message has to be identical in both cases? this doesn't sound logical to me. two different situations, why the message is the same?